tutorials/fit/combinedFit.py

```diff
@@ -122,14 +122,16 @@ def __call__(self, par):
 fB.SetFitResult(result, iparB)
 fB.SetRange(rangeB().first, rangeB().second)
 fB.SetLineColor(ROOT.kBlue)
-hB.GetListOfFunctions().Add(fB)
+# The function list takes ownership of the added elements, so we are cloning
+# the function to a double delete.
+hB.GetListOfFunctions().Add(fB.Clone())
```

Comment:
A bit ugly, but understandable with the interfaces involved

Reply:
An alternative would be to do `ROOT.SetOwnership(fB, False)`, but this might confuse the novice user even more :(